Pod/Classes/GimbalAirshipAdapter.swift

```diff
@@ -87,25 +88,28 @@ fileprivate let shouldTrackRegionEventsKey = "gmbl_should_track_region_events"
      */
     @objc open var shouldTrackRegionEvents : Bool {
         get {
-            return UserDefaults.standard.bool(forKey: shouldTrackRegionEventsKey)
+            return defaults.bool(forKey: shouldTrackRegionEventsKey)
         }
         set {
-            UserDefaults.standard.set(newValue, forKey: shouldTrackRegionEventsKey)
+            defaults.set(newValue, forKey: shouldTrackRegionEventsKey)
         }
     }
     
+    @objc private let defaults: UserDefaults
+    
     #if !targetEnvironment(simulator)
     private override init() {
         placeManager = PlaceManager()
-        gimbalDelegate = AirshipGimbalDelegate()
+        defaults = UserDefaults(suiteName: defaultsSuiteName) ?? UserDefaults.standard
```

Comment:
Under what circumstances does the `standard` defaults get used? In this case, I think `migrateDefaults()` will just delete data?

Reply:
As I understand it, the only situation where an init of a Defaults suite fails (which is the case where `standard` gets used) would be if you were to use your own bundle ID, or the `global domain`, as suite names.

Reply:
Got it, sounds good.